examples/test-LwIP-netconn/scpi_server.c

```diff
@@ -253,6 +253,10 @@ static int processIoListen(user_data_t * user_data) {
         } else {
             /* connection established */
             iprintf("***Connection established %s\r\n", inet_ntoa(newconn->pcb.ip->remote_ip));
+            newconn->pcb.tcp->so_options |= SOF_KEEPALIVE;
```

Comment:
Probably `ip_set_option` should be used as lwip sintactic sugar and not directly accessing `so_options`. Same for control connection.

```c
ip_set_option(newconn->pcb.tcp, SOF_KEEPALIVE);
```